### ISSUE TITLE
omf update returns status 1

### ISSUE THREAD

**Issue by rominf** [fish-shell/oh-my-fish#486]
```
 omf update
Updating extract... ✔
Updating gi... ✔
Updating localhost... ✔
Updating pyenv... ✔
Updating vi-mode... ✔
Updating bak... ✔
Updating dpaste... ✔
Updating bobthefish... ✔
 echo $status
1
```

**Comment by mandx** [comment]
Same here. How can we find out which command failed?

**Comment by bobthecow** [comment]
[It's this statement](https://github.com/oh-my-fish/oh-my-fish/blob/master/plugins/omf/omf.packages.fish#L40-L42). As I see it, there are two problems:

 * Because the if is (always) false, `$status` is false.
 * Because the developer of the `omf` plugin doesn't use an awesome theme like bobthefish, he didn't see the incorrect status :)

**Comment by bpinto** [comment]
This has now been fixed with #512
